includes/blocks/class-sensei-course-progress-block.php

```diff
@@ -51,11 +51,10 @@ public function render_course_progress( $attributes ) : string {
 			return '';
 		}
 
-		list(
```

Comment:
I did a refactor in the places we were using the `list` with keys to still keep the [support to PHP 7.0](https://www.php.net/manual/en/function.list.php).

Reply:
Great! Didn't know this key-based approach even existed. Much better with the array keys in my opinion 💪